addons/phone_validation/models/models.py

```diff
@@ -48,7 +69,7 @@ def _phone_format(self, fname=False, number=False, country=False, force_format='
           returned.
         """
         if not number:
-            # if no number is given, having a singletong recordset is mandatory to
+            # if no number is given, having a singleton recordset is mandatory to
```

Comment:
Was it blocytong maybe ?

Reply:
:bricks: oh, thought I changed that line, guess not